src/utils/database.ts

```diff
@@ -0,0 +1,40 @@
+import { existsSync, readFileSync, writeFileSync, unlinkSync } from 'fs';
+
+export class Database {
+  defaultFilePath: string;
+
+  constructor() {
+    this.defaultFilePath = 'src/utils/data.json';
+  }
+
+  async hasGameFile(filePathOverride?: string): Promise<boolean> {
+    return existsSync(filePathOverride || this.defaultFilePath);
```

Comment:
Sync functions aren't asynchronous so these promises are unnecessary. I brought up sync functions in planning so that we don't have to work with promises in the data layer for now.